package.json

```diff
@@ -40,7 +40,8 @@
         "eslint-plugin-security": "1.5.0",
```

Comment:
two minor things before merging:
`npm run bench` and update package* files by running `npm install`.
and squash commits so I proceed with the merge

Reply:
Ah, doh, I missed the benchmarks! 
The CI doesn't seem to run unless triggered manually?

Anyway, I fixed the benchmark references to Group, ran npm install, and squashed all the commits. 

Thanks :)